### ISSUE TITLE
Alpha 15 loses key pressed effect

### ISSUE THREAD

**Issue by jvaleskadevs** [Askannz/msi-perkeyrgb#44]
Hi!

I have a MSI Alpha 15 laptop, I 'm able to change the color in steady mode and use the presets but the problem is that my laptop used to have an effect that WHEN PRESSING A KEY IT CHANGES THE COLOR. Now, color not changes when pressing a key.

EDIT: After reading [https://github.com/Askannz/msi-perkeyrgb/issues/24](https://github.com/Askannz/msi-perkeyrgb/issues/24)
I think that if use the next config.cfg file in tag v2.0 I'm been able to restore my default configuration but I not have clear if this could brick my laptop keyboard: 

all steady ff0000
all effect keypressed
 
effect keypressed
         start ff0000
         trans ff00ff 100
         trans ff0000 200
end

Is there a way to recover the default settings? or to apply this effect currently with msi-perkeyrgb command line tool?

Which model I must select in the models list since Alpha 15 is not in the list?

Thanks

**Comment by vwbusguy** [comment]
I'm in the same boat with my MSI Alpha 15.

**Comment by vwbusguy** [comment]
It *looks* like it would be fine since you're including transitions, but I'm just not feeling brave enough to risk it.  I wish I'd see this before I used this program and lost that ability, though.

**Comment by jvaleskadevs** [comment]
Indeed, the risk outweighs my need, although I would sincerely like to recover that effect.

On the other hand, have you tried in Windows with the official MSI program, I don't have Windows and don't intend to install it but if you have dual boot maybe you can restore it to its original state.

**Comment by vwbusguy** [comment]
I do not have a dual boot with Windows.  I'm only running Linux, unfortunately.  That said, it be great if SteelSeries fixed the problem in #24  and released the firmware via lvfs.
